soh/soh/Enhancements/randomizer/3drando/starting_inventory.cpp

```diff
@@ -55,7 +55,9 @@ void GenerateStartingInventory() {
     AddItemToInventory(SHADOW_TEMPLE_BOSS_KEY);
   }
 
-  if (GanonsBossKey.Is(GANONSBOSSKEY_START_WITH)) {
+  // Add Ganon's Boss key with Triforce Hunt so the game thinks it's obtainable from the start.
+  // During save init, the boss key isn't actually given and it's instead given when completing the triforce.
+  if (GanonsBossKey.Is(GANONSBOSSKEY_START_WITH) || GanonsBossKey.Is(GANONSBOSSKEY_TRIFORCE_HUNT)) {
```

Comment:
what happens if we don't do this?

Reply:
Ganon's Boss key would be placed in the item pool on top of also being granted by collecting the required amount of triforce pieces.